modelon/impact/client/operations.py

```diff
@@ -23,9 +25,23 @@ class Status(Enum):
     DONE = "done"
 
 
-class Operation(ABC):
+@enum.unique
+class UploadStatus(Enum):
```

Comment:
This one is actually a bit more generall than `upload` for example we have this status also for workspace export. So I suggest calling this status 'AsyncOperationStatus'.